modules/adding-to-a-project.adoc

```diff
@@ -7,9 +7,12 @@
 
 .Procedure
 
-. Navigate to *Home* -> *Projects*.
+. Select *Developer* from the context selector at the top of the web console
+navigation menu.
 
-. Select a project.
+. Click *+Add*
 
-. In the upper right-hand corner of the *Project Status* menu, click *Add*, then
-choose from the provided options.
+. At the top of the page, select the name of the project that you want to add to.
```

Comment:
I don't spend enough time in the UI (I don't have it in front of me now) -- Is it possible to be more specific about where the project name is? Or is it just an un-labeled UI element?

Reply:
It's unlabeled